recipes/cuda-python/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
+{% set patch_version = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - setuptools
+    - cython
+    - pycparser
+    - autopxd2
+    - pyclibrary
+    - python
+    - pip
+    - cudatoolkit {{ major_version ~ ".*.*" }}
+    - pywin32 # [win]
```

Think this is what the linter is referring to. Idk if that is also related to [the CI failure]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=377887&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77 ), but worth trying for that reason as well

```suggestion
    - pywin32  # [win]
```